build-with-docker.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p out
+docker build -t maruos/blueprints .
+docker run --privileged --rm \
+	-v /var/cache:/var/cache \
+        -v `pwd`:/var/maru \
+	-ti maruos/blueprints /bin/bash
```

Comment:
I have an idea to avoid starting a separate shell within the container: please see my comments on the PR.